# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/tests/css/css-grid/subgrid/line-names-001.html
# Commit: 2ac943ae4424
# Full Hash: 2ac943ae442445f96f5a68d5298b320f30b1313a
# Author: Mats Palmgren <mats@mozilla.com>
# Date: 2019-10-13 09:36:01
# Regressor Bug: 1234311
# File Overlap Count: 1
# Description:
#   Bug 1234311 part 2 - [css-grid-2] Implement 'grid-template-rows/columns' resolved value correctly for subgrid.  r=dholbert
#   
#   This implements the resolution and adds WPTs for:
#   https://github.com/w3c/csswg-drafts/issues/4362
#   
# ==============================================================================

diff -r 43f18d905d75 -r 2ac943ae4424 testing/web-platform/tests/css/css-grid/subgrid/line-names-001.html
--- a/testing/web-platform/tests/css/css-grid/subgrid/line-names-001.html	Sat Oct 12 17:15:43 2019 +0000
+++ b/testing/web-platform/tests/css/css-grid/subgrid/line-names-001.html	Sat Oct 12 17:16:02 2019 +0000
@@ -38,5 +38,20 @@
   </div>
 </div>
 
+<script>
+  const expectedResults = [
+    "subgrid [] [] [] []",
+  ];
+  [...document.querySelectorAll('.subgrid')].forEach(function(subgrid, i) {
+    let actual = window.getComputedStyle(subgrid)['grid-template-columns'];
+    let expected = expectedResults[i];
+    if (actual != expected) {
+      let err = "Unexpected getComputedStyle value for subgrid " + i + " with className '" + subgrid.className + "':" +
+      " Actual: \"" + actual + "\", Expected: \"" + expected + "\"";
+      document.body.appendChild(document.createTextNode(err));
+    }
+  });
+</script>
+
 </body>
 </html>